CHANGELOG.md

```diff
@@ -1,40 +0,0 @@
-# <a name="changelog"></a>CHANGE LOG
```

Comment:
Not following - do you mean you re-added it? If so, I don't see it in the commits so far :)